From 217805a03a4fcd050b40fb25daf6246c73fc795c Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Thu, 6 Sep 2018 21:27:28 +0000
Subject: [PATCH 14/14] 4.18-Backport RPi 3B Plus ethernet dtsi

Backport bcm283x-rpi-lan7515.dtsi RPi 3B Plus ethernet dtsi from 4.18.6

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi | 27 ++++++++++++++++++++++
 1 file changed, 27 insertions(+)
 create mode 100644 arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi

diff --git a/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
new file mode 100644
index 00000000..9403da09
--- /dev/null
+++ b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0
+/ {
+	aliases {
+		ethernet0 = &ethernet;
+	};
+};
+
+&usb {
+	usb-port@1 {
+		compatible = "usb424,2514";
+		reg = <1>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		usb-port@1 {
+			compatible = "usb424,2514";
+			reg = <1>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			ethernet: ethernet@1 {
+				compatible = "usb424,7800";
+				reg = <1>;
+			};
+		};
+	};
+};
-- 
2.17.1

